pywr/parameters/_parameters.pyx

```diff
@@ -992,6 +1016,19 @@ cdef class RbfProfileParameter(Parameter):
     cpdef double[:] get_double_upper_bounds(self):
         return self._upper_bounds
 
+    # Integer variables are for the days of the year positions (if optimised)
+    cpdef set_integer_variables(self, int[:] values):
+        self.days_of_year = [1] + np.array(values).tolist()
```

Comment:
This is a good spot. I think the should be checked on `__init__` rather than adjusting the variables in `set_integer_variables`. Or do you think you should allow this to be possible to declare, but any out of order values be made to work (either by reordering or capping to have at least 1 day gap)?

Reply:
might be cleaner and more understandable to have it checked in `__init__` but on the other hand adjust the variables in `set_integer_variables` would give users a little more flexibility when initialising the class. I don't really have a strong preference either way.

Reply:
Done. I did it in `__init__` and also checked for strictly monotonic days of year.